students/liikt/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"bufio"
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+func sanitize(s string) string {
+	return strings.ToLower(strings.Trim(s, "\n\r\t"))
+}
+
+func main() {
+	problemsFile := flag.String("path", "problems.csv", "This is the flag to the CSV containing the problems for the quiz")
+	timeout := flag.Int("timeout", 30, "The amount of time you have for a single question in seconds")
+	flag.Parse()
+
+	file, err := os.Open(*problemsFile)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer file.Close()
+
+	problems, err := csv.NewReader(file).ReadAll()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	correct := 0
```

Comment:
Although this is totally fine and there is nothing wrong with this, it is prefered to initialise a variable this way:
```go
var (
        correct int
        b bool
)
```